02_configure_host.sh

```diff
@@ -54,6 +54,18 @@ if [[ ! -z "${MIRROR_IMAGES}" || $(env | grep "_LOCAL_IMAGE=")  || ! -z "${ENABL
     setup_local_registry
 fi
 
+# Configure a local proxy to be used for the installation
+if [[ ! -z "${INSTALLER_PROXY}" ]]; then
+  generate_proxy_conf > squid.conf
+
+  sudo podman run -d --rm \
+     --net host \
+     --volume $PWD/squid.conf:/etc/squid/squid.conf \
+     --name ds-squid \
+     --dns 127.0.0.1 \
+     quay.io/sameersbn/squid:latest
```

Comment:
This image was last updated 3 years ago, is there a reason this was chosen or is there a more appropriate (maintained) upstream/downstream image we can use?

Reply:
AFAICS it's an old/unmaintained image e.g see https://github.com/sameersbn/docker-squid and compare the squid version with the latest stable one - we don't have to block this PR on it, but IMO we should look at building an updated version, or find an actively maintained alternative?

Reply:
Agree, as long as we keep in sync with the CI to avoid the burden of maintaining multiple version. Will look after for something newer.